docs/cloud/pouta/tutorials/image-pipeline.md

```diff
@@ -0,0 +1,650 @@
+| name              | None                                                                                                                                                                       |
+| port_range_max    | 22                                                                                                                                                                         |
+| port_range_min    | 22                                                                                                                                                                         |
+| project_id        | ef20bcc4215d49ddbf33e5c5740d89fe                                                                                                                                           |
+| protocol          | tcp                                                                                                                                                                        |
+| remote_group_id   | None                                                                                                                                                                       |
+| remote_ip_prefix  | 0.0.0.0/0                                                                                                                                                                  |
+| revision_number   | 0                                                                                                                                                                          |
+| security_group_id | a8630776-db3d-408a-ba8e-8c52b5f2a8c9                                                                                                                                       |
+| tags              | []                                                                                                                                                                         |
+| updated_at        | 2025-02-04T13:13:04Z                                                                                                                                                       |
++-------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+```
+
+Now we apply the security group to the previously created virtual machine, so that this newly-created rule applies to its traffic.
+```
+$ openstack server add security group pipeline_vm pipeline_security_group
+```
+In case of success, the command will show no output.
+
+### Connecting to the virtual machine
+
+The virtual machine is now configured to allow traffic from our workstation but it is not reachable yet because it has no address assigned to it.
+
+We go back to `terminal_C`.
```

Comment:
I have already forgotten which `terminal_C` is maybe name them `terminal_pouta` , `termianl_pukki` and `terminal_allas` .

Reply:
I agree

Reply:
Fixed in 16c117e3673ddafd9a10ff585ecf378bfd1b06db